rlp/Cargo.toml

```diff
@@ -5,12 +5,15 @@ description = "Recursive-length prefix encoding, decoding, and compression"
 repository = "https://github.com/paritytech/parity-common"
 license = "MIT/Apache-2.0"
 authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
 
 [dependencies]
-byteorder = "1.0"
-rustc-hex = {version = "2.0", default-features = false }
-
-[dev-dependencies]
-hex-literal = "0.1"
```

Comment:
Why prefer `rustc-hex` to `hex-literal`?

Reply:
I think `rustc-hex` is enough to encode/decode hex. There is no need to introduce more dependencies, even if it is just `dev-dependencies`.

Reply:
Tests need to be as readable as possible, which is why I think`hex-literal` is better.

Reply:
(The new `0.2` of `hex-literal` is edition compatible too!)